src/ripple/test/jtx/Env.h

```diff
@@ -377,7 +456,7 @@ class Env
 
     /** Return the TER for the last JTx. */
     TER
-    ter() const
+    ter()
```

Comment:
I understand that C++-11 has a stricter definition of `const` than previous versions. However, most read-only functions should still be marked as `const`. Many of the functions here should still be `const`. By adding an `Application::openLedger() const` function we can bring back `const`. Here's a branch that shows what I have in mind: https://github.com/ripple/rippled/commit/52e92331c155857dedb2cb0c92fd46ae9388b6dc